rpc/comms/ipc_windows.go

```diff
@@ -698,27 +698,27 @@ func startIpc(cfg IpcConfig, codec codec.Codec, initializer func(conn net.Conn)
 	os.Chmod(cfg.Endpoint, 0600)
 
 	go func() {
+		defer os.Remove(cfg.Endpoint)
```

Comment:
The amount of return values of this function needs to change as well (not on this line, L691)